packages/wallet/ui/src/components/ProvisionDialog.jsx

```diff
@@ -24,9 +26,52 @@ const errors = {
 // TODO: Read this from the chain via rpc.
 const CREATION_FEE = '10 BLD';
 
-const ProvisionDialog = ({ onClose, open, address, href, keplrConnection }) => {
+// 100 IST
+const MINIMUM_PROVISION_POOL_BALANCE = 100_000_000n;
+
+/** @typedef {import('@agoric/vats/src/provisionPool.js').MetricsNotification} ProvisionPoolMetrics */
+
+export const useProvisionPoolMetrics = (unserializer, leader) => {
+  const [data, setData] = useState(/** @type {ProvisionPoolMetrics?} */ (null));
+
+  useEffect(() => {
+    let cancelled = false;
+    const fetchData = async () => {
+      const follower = await makeFollower(
+        `:published.provisionPool.metrics`,
+        leader,
+        {
+          unserializer,
+        },
+      );
+      for await (const { value } of iterateEach(follower)) {
```

Comment:
I think we want `iterateLatest`. The intermediate states don't matter

Reply:
Done, thanks for catching!